conf/logs.json

```diff
@@ -308,5 +308,524 @@
         }
       }
     }
-  }
-}
\ No newline at end of file
+  },
+  "bro:conn.log": {
+    "parser": "json",
+    "schema": {
+      "ts": "string",
+      "uid": "string",
+      "id": "string",
+      "proto": "string",
+      "service": "string",
+      "duration": "string",
+      "orig_bytes": "integer",
+      "resp_bytes": "integer",
+      "conn_state": "string",
+      "local_orig": "string",
+      "local_resp": "string",
+      "missed_bytes": "integer",
+      "history": "string",
+      "orig_pkts": "integer",
+      "resp_ip_bytes": "integer",
+      "tunnel_parents": "string",
+      "orig_l2_addr": "string",
+      "resp_l2_addr": "string",
+      "vlan": "integer",
+      "inner_vlan": "integer"
+    }
+  },
+  "bro:dce_rpc.log": {
+    "parser": "json",
+    "schema": {
+      "ts": "string",
+      "uid": "string",
+      "id": "string",
+      "rtt": "string",
+      "named_pipe": "string",
+      "endpoint": "string",
+      "operation": "string"
+    }
+  },
+  "bro:dhcp.log": {
+    "parser": "json",
+    "schema": {
+      "ts": "string",
+      "uid": "string",
+      "id": "string",
+      "mac": "string",
+      "assigned_ip": "string",
+      "lease_time": "integer",
```

Comment:
`lease_time` is of type `interval`: https://www.bro.org/sphinx/script-reference/types.html#type-interval. The documentation shows that the value doesn't need to be an integer, it can be a string.

This means any field that is of type `interval` must be labelled as a `string`. Make this change for any field of type `interval`